### PR TITLE
Switch to using a team assignment for changelogs

### PR DESCRIPTION
**Pull request by calebmshafer**
The first cut of the codeowners specified each reviewer individually. This has lead to a large number of reviewers on every PR.

To try and reduce this I've updated many of the sections with GH Teams instead.  This will allow us to manage who is added based on [review assignment](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/managing-code-review-assignment-for-your-team#routing-algorithms) which will automatically pick 1-2 people (can be configured) from each team to be added to a PR.

I setup the `@imodeljs/admins` Team to round robin 2 assignees and then it will be up to those 2 people to either review/approve or assign it to the correct person to review a change.

A couple of things to keep in mind;
1. Just because someone in the team is not added as a reviewer does not mean they cannot review and approve
1. Not everyone on the team will be notified of the PR, only the individuals add (this can be configured)

The following teams were already in place so I used them initially (we can always create more if necessary),

1. `@imodeljs/devs`
1. `@imodeljs/admins`
1. `@imodeljs/core`
1. `@imodeljs/display`
1. `@imodeljs/presentation`
1. `@imodeljs/ui` 

> Note: I have only configured the round robin style assignment for `@imodeljs/admins`, if anyone else wants to configure it and doesn't know how please let me know.

@mgooding @NancyMcCallB @DanEastBentley @bsteinbk @grigasp, please take a read through the above since it directly impacts how reviewers work and you're member and/or admins of some of the teams.

### CODE REVIEW

**Comment by scsewall** [comment]
The comment and the team mentioned don't match

**Review by ColinKerr — APPROVED** [review]

**Comment by calebmshafer** [comment]
> The comment and the team mentioned don't match

@scsewall based on the documentation for adding a team as a reviewer, I thought that if I added the high level "devs" team and just set it so no one gets notified it would work.  However, I did some testing in a test GitHub Org and it doesn't seem to be working the way I want it to be.  I'll do some more testing and figure out a different way since it's annoying the way it's currently setup.

**Comment by calebmshafer** [comment]
> The comment and the team mentioned don't match

@scsewall just took another swing at it.  Mind taking a look?

**Comment by grigasp** [comment]
I removed myself as a code owner from bentleyjs-core and ui. I hope @imodeljs/ui team don't mind having a look at our ui-related changes from time to time.

**Review by DanEastBentley — APPROVED** [review]

**Review by bsteinbk — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -157,10 +144,7 @@ rush.json @calebmshafer @scsewall
...
-/ui/components/src/ui-components/properties   @bsteinbk @DanEastBentley @grigasp @saskliutas
-/ui/components/src/ui-components/propertygrid @bsteinbk @DanEastBentley @grigasp @saskliutas
-/ui/components/src/ui-components/tree         @bsteinbk @DanEastBentley @grigasp @saskliutas
-# /ui/components/src/ui-components/properties   @bsteinbk @DanEastBentley @grigasp @saskliutas @roluk
-# /ui/components/src/ui-components/propertygrid @bsteinbk @DanEastBentley @grigasp @saskliutas @roluk
-# /ui/components/src/ui-components/tree         @bsteinbk @DanEastBentley @grigasp @saskliutas @roluk
+/ui @imodeljs/ui @grigasp
+/ui/components/src/ui-components/properties   @bsteinbk @DanEastBentley @imodeljs/presentation
+/ui/components/src/ui-components/propertygrid @bsteinbk @DanEastBentley @imodeljs/presentation
+/ui/components/src/ui-components/tree         @bsteinbk @DanEastBentley @imodeljs/presentation
```
[the remark this answers is not in the document]
Comment (grigasp) [reply .github/CODEOWNERS]: Changed already

**Review by scsewall — APPROVED** [review]

**Pull request merged**